### DISCUSSION TITLE
DZBB-TV

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#663] [Q&A]
**Channel Name:** GMA 7
**Channel ID:** DZBBDT1.ph
**Country:** Philippines
**Language:** Tagalog
**External Link (optional):** xxx
**Notes (optional):** Free to air channel in the Philippines.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/ph.m3u#L22-L23
